src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -345,4 +345,7 @@
   <data name="ToggleRetroEffectCommandKey" xml:space="preserve">
     <value>Toggle retro terminal effect</value>
   </data>
+  <data name="BreakIntoDebuggerCommandKey" xml:space="preserve">
+    <value>Break into the debugger</value>

```

Comment:
feel free to localization lock this entire phrase -- no need to have them deal w/ it, yeah?

Reply:
sure, I'll bump it

Reply:
i decided it was ok to ship like this